hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -275,10 +325,18 @@ genCloseMutation (tx, _utxo) =
     , SomeMutation (Just $ toErrorCode ChangedParameters) MutateHeadIdInOutput <$> do
         otherHeadId <- toPlutusCurrencySymbol . headPolicyId <$> arbitrary `suchThat` (/= Fixture.testSeedInput)
         pure $ ChangeOutput 0 $ changeHeadOutputDatum (replaceHeadId otherHeadId) headTxOut
-    , SomeMutation Nothing MutateRequiredSigner <$> do
-        newSigner <- verificationKeyHash <$> genVerificationKey
+    , SomeMutation (Just $ toErrorCode SignerIsNotAParticipant) MutateRequiredSigner <$> do
+        newSigner <- verificationKeyHash <$> genVerificationKey `suchThat` (/= somePartyCardanoVerificationKey)
```

Comment:
I've found that this was changed in commit [4eaafec002](https://github.com/input-output-hk/hydra/commit/4eaafec002):

> Add first round of label descriptions for close mutations

I would split this commit to only include label descriptions and put this change in a dedicated commit which would explain why it is ok to add `suchThat (/= somePartyCardanoVerificationKey)`

Reply:
this is not actually needed, as the chances to have this test produce a verification key equal to the healthy one is very unlikely.

anyhow, I added this condition to improve the mutation readability and make it explicit what is the intention.

do you like it or do you prefer to get rid of this?